packages/tenderly-core/src/internal/cli/commands/LoginCommand.ts

```diff
@@ -2,39 +2,49 @@ import open from "open";
 import axios from "axios";
 import prompts from "prompts";
 import commander from "commander";
+import { logger } from "../../../utils/logger";
 
 import { isAccessTokenSet, setAccessToken } from "../../../utils/config";
 import { TENDERLY_API_BASE_URL, TENDERLY_DASHBOARD_BASE_URL } from "../../../common/constants";
 
 export const LoginCommand = new commander.Command("login").description("login to Tenderly").action(async () => {
+  logger.info("Trying to login to Tenderly...");
+
   if (isAccessTokenSet()) {
+    logger.debug("Access token is already set. Checking if user wants to overwrite it with a new one...");
     const response = await prompts({
       type: "confirm",
       name: "overwrite",
       message: "Access token already set. Would you like to overwrite it?",
     });
     if (!response.overwrite) {
+      logger.debug("User didn't request an overwrite of the token. Proceeding logging in with existing token.");
       return;
     }
   }
 
+  logger.info("Access token isn't set. Waiting for user to provide it...");
   const accessToken = await promptAccessToken();
 
+  logger.debug("User access token accepted. Trying to log in...");
```

Comment:
Nitpick: I'm not sure how much I like using this type of message everywhere, as a user when running an application in debug mode it would be super weird for me to see this third person addressing:

```bash
User access token accepted. Trying to log in....

User trying to xyz...
```

Reply:
I guess this makes sense. Better approach would be to just say `Access token accepted`, `Trying to xyz...`